packages/vue/src/components/organisms/SfFooter/SfFooter.vue

```diff
@@ -12,22 +24,49 @@ import {
   mapMobileObserver,
   unMapMobileObserver,
 } from "../../../utilities/mobile-observer";
+import SfLink from "../../atoms/SfLink/SfLink";
+import SfImage from "../../atoms/SfImage/SfImage";
+
 Vue.component("SfFooterColumn", SfFooterColumn);
 export default {
   name: "SfFooter",
+  components: { SfLink, SfImage },
   props: {
+    /**
+     * Number of footer columns
+     */
     column: {
       type: Number,
       default: 4,
     },
+    /**
+     * Multiple footer columns open at the same time on mobile
+     */
     multiple: {
       type: Boolean,
       default: true,
     },
+    /**
+     * Footer columns open on mobile
+     */
     open: {
       type: [String, Array],
       default: () => [],
     },
+    /**
+     * Footer logo
+     */
+    logo: {
```

Comment:
I don't think we should expand footer about logo prop :)

Reply:
Why do we need `logo` as a prop here?